react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = { topicId: '' };
+  }
+
+  handleClick(number) {
+    const topicId = this.state.topicId;
+
+    if (topicId === '') {
```

Comment:
Since the first `if` and `else` perform the same action you can simplify this a bit with:

```JAVASCRIPT
if (topicId === number) {
  this.setState({ topicId: '' });
} else {
  this.setState({ topicId: number });
}
```